src/include/optimizer/cost_and_stats_calculator.h

```diff
@@ -40,7 +40,9 @@ class CostAndStatsCalculator : public OperatorVisitor {
 
   inline double GetOutputCost() { return output_cost_; }
 
-  void Visit(const PhysicalScan *) override;
+  void Visit(const DummyScan *) override;
```

Comment:
This is the Optimizer team's code(which has not yet been merged) which we needed to get our code working.